rules/github-scrum-workflow/rule.md

```diff
@@ -0,0 +1,79 @@
+---
+type: rule
+archivedreason:
+title: How to set up GitHub scrum workflow
+guid: e63da194-c5d2-4830-9106-d164aafcddb5
+uri: do-you-know-the-how-to-set-up-github-workflow
+created: {date_created}
+authors: []
+related:
+
+---
+### How to set up GitHub scrum workflow
+
+There are several ways of turning your GitHub repository into a simple but powerful agile work environment, the following is the recommended:
```

Comment:
I think we should start the rule out by saying what the issue is with using GitHub in it's vanilla state with a bad example. I would also explain how Azure DevOps has a solution to this but for GitHub we want to stay inside GitHub. Then we can present a few options (find the most popular ones), their strengths/weaknesses and follow the rule: https://www.ssw.com.au/rules/do-you-manage-up by providing several options with one recommended option.

Then we could end with a good example.

Let me know your thoughts

Reply:
Hi @pierssinclairssw , I do agree that we should include this to give more context. Let me know your thoughts on this:

> GitHub in its vanilla state is not well tailored for a Scrum workflow which does not allow Product Backlog Items (PBIs) and Sprint tracking like other version control system such as Azure DevOps. However, we prefer to stay inside GitHub as it is fast, easy to use and inexpensive. There have been a few options to turn GitHub into an agile work environment but the following is the most recommended:

Reply:
> GitHub in its vanilla state is not well tailored for the Scrum workflow because it does not allow Product Backlog Items (PBIs) to be tracked in a Sprint like other version control system such as Azure DevOps. However, we prefer to stay inside GitHub as it is fast, easy to use and inexpensive. There are a few options to turn GitHub into an agile work environment but the following is the most recommended:

I would also list out the other options and then explain the recommended approach (we don't need to go into detail about other approaches).

Reply:
Hey @pierssinclairssw 

Done - changed introduction with some other options added

> GitHub in its vanilla state is not well tailored for the Scrum workflow because it does not allow Product Backlog Items (PBIs) to be tracked in a Sprint like other version control system such as Azure DevOps. However, we prefer to stay inside GitHub as it is fast, easy to use and inexpensive. There are a few options to turn GitHub into an agile work environment such as using GitHub Projects and Labels but the following is the most recommended: